mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -169,11 +177,20 @@
         <CmdButtonArray index="Execute">
             <Flags index="ShowInGlossary" value="0"/>
         </CmdButtonArray>
+        <InfoArray index="0">
+            <SectionArray index="Stats">
+                <DurationArray index="Duration" value="0"/>
```

Comment:
**undocumented**: Infestor's Unburrow stats transition duration is now changed from 0.5 to 0
Does this cause unburrowing units to become "visible" immediately instead of after ~350ms?